srv/wordpress/wp-content/plugins/acore-wp-plugin/src/Hooks/WooCommerce/CartValidation.php

```diff
@@ -15,6 +15,7 @@ class CartValidation extends \ACore\Lib\WpClass {
         "char-restore-delete" => ['acore_char_sel'],
         "char-transfer-sku" => ['acore_char_sel', 'acore_dest_account'],
         "itemsend" => ['acore_char_sel'],
+        "guild-rename" => ['acore_char_sel', 'acore_new_guild_name'],
```

Comment:
when you add a new service, add the SKU in the documentation https://github.com/azerothcore/acore-cms/blob/master/docs/configure-cms.md